source/PreferencesPanel.cpp

```diff
@@ -489,7 +414,8 @@ void PreferencesPanel::DrawControls()
 			bool isHovering = (index == hover && !isEditing);
 			if(!isHovering && index == selected)
 			{
-				table.SetHighlight(-120, 54);
+				auto textWidth = FontSet::Get(14).Width(command.Description());
+				table.SetHighlight(-120, textWidth - 110);
```

Comment:
:pray:
That annoyed me. Thank you for fixing it.
![image](https://github.com/endless-sky/endless-sky/assets/17688683/19c8085a-634e-4077-a169-d66eba6ca828)
![image](https://github.com/endless-sky/endless-sky/assets/17688683/d94c7ed0-e879-4187-990f-809d9ff85e54)